docs/databases/connections/google-analytics.md

```diff
@@ -4,112 +4,13 @@ redirect_from:
-
-- **Choose when syncs and scans happen**: When this setting is disabled (which it is by default), Metabase regularly checks the database to update its internal metadata. If you have a large database, we you can turn this on and control when and how often the field value scans happen.
-
-- **Periodically refingerprint tables**: This setting — disabled by default — enables Metabase to scan for additional field values during syncs allowing smarter behavior, like improved auto-binning on your bar charts.
-
-Please see the [database sync and analysis documentation][sync-docs] for more details about these toggle settings.
-
-## Saving your database configuration
-
-When you're done, click the **Save** button. A modal dialog will inform you that your database has been added. You can click on **Explore this data** to see some automatic explorations of your data, or click **I'm good thanks** to stay in the **Admin Panel**.
-
-## Checking if Google Analytics is working correctly
-
-Give Metabase [some time to sync][sync-docs] with your Google Analytics dataset, then exit the **Admin Panel**, click on **Browse Data**, find your Google Analytics database, and start exploring. Once Metabase is finished syncing, you will see the names of your Properties & Apps in the data browser.
-
-If you're having trouble, see the guides under [Troubleshooting data sources][troubleshooting-data-sources].
-
-[google-analytics]: https://cloud.google.com/analytics
-[google-analytics-add-user]: https://support.google.com/analytics/answer/1009702
-[google-api-overview]: https://console.cloud.google.com/apis/api/analytics.googleapis.com/overview
-[google-cloud]: https://cloud.google.com/
-[google-cloud-create-project]: https://cloud.google.com/resource-manager/docs/creating-managing-projects#creating_a_project
-[google-cloud-management]: https://cloud.google.com/resource-manager/docs/creating-managing-projects
-[google-cloud-platform-console]: https://console.cloud.google.com/
-[google-cloud-oauth]: https://support.google.com/cloud/answer/6158849
-[google-enable-disable-apis]: https://support.google.com/googleapi/answer/6158841
-[google-managing-service-accounts]: https://cloud.google.com/iam/docs/creating-managing-service-accounts
-[google-oauth-scopes]: https://developers.google.com/identity/protocols/oauth2/scopes
-[google-service-accounts]: https://cloud.google.com/iam/docs/service-accounts
-[sync-docs]: ../connecting.md#syncing-and-scanning-databases
-[troubleshooting-data-sources]: ../../troubleshooting-guide#databases
+- Upgrading to the latest version of [Google Analytics](https://support.google.com/analytics/answer/10089681?hl=en&ref_topic=12154439,12153943,2986333).
+- Setting up a [BigQuery](https://cloud.google.com/bigquery) account.
+- Connecting your [Bigquery to Metabase](./bigquery.md).
+- Exporting your [Google Analytics 4 data to BigQuery](https://support.google.com/analytics/answer/9358801?hl=en).
```

Comment:
We should also add that, as GA will bring the data to BQ in 1 table per day, users are encouraged to build native SQL models with wildcard queries (https://cloud.google.com/bigquery/docs/querying-wildcard-tables) and then build the questions over those views